lib/replace-vars.js

```diff
@@ -1,6 +1,33 @@
-module.exports = (input, source) => {
+module.exports = {
+  replace,
+  replaceUrlPartialChunk,
+};
+
+function replace(input, source) {
   return (input || '').replace(/(\${.*?})/g, (_, match) => {
     const key = match.slice(2, -1); // ditch the wrappers
     return source[key] || '';
   });
-};
+}
+
+function replaceUrlPartialChunk(chunk, prevPartial, config) {
+  const replaceRegExp = new RegExp(config.RES_BODY_URL_SUB, 'g');
```

Comment:
Think it's OK to import `config` here instead passing as a parameter.

Reply:
Should relay.js also import config?

Reply:
This is config is created and then passed through the broker, see: https://github.com/snyk/broker/blob/551f750137c39b74922245b7c00613c576525b9f/lib/index.js#L20